File: api/src/main/java/fr/uha/ensisa/stegmiller/appintav/api/dto/response/UserEventsResponseDto.java
package fr.uha.ensisa.stegmiller.appintav.api.dto.response;

import fr.uha.ensisa.stegmiller.appintav.api.dto.model.ReducedEventDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserEventsResponseDto {
    Iterable<ReducedEventDto> eventsOrganized;

    Iterable<ReducedEventDto> events;
}

File: front/src/shared/dtos/response/user-events-response.dto.ts
import {Expose, Type} from "class-transformer";
import {EventReducedDto} from "../event-reduced.dto";

export class UserEventsResponseDto {

    @Expose()
    @Type(() => EventReducedDto)
    eventsOrganized?: EventReducedDto[];

    @Expose()
    @Type(() => EventReducedDto)
    events?: EventReducedDto[];

}
